tests/test_utils.py

```diff
@@ -868,6 +868,65 @@ def test_find_substitution():
     assert assignment == {var3: plate0, var0: cup0}
 
 
+def test_LinearChainParameterizedOption():
```

Comment:
also test what happens if empty list is passed in for children, and what happens if the options don't chain correctly